src/databricks/labs/ucx/workspace_access/redash.py

```diff
@@ -73,7 +73,7 @@ def _is_item_relevant(item: Permissions, migration_state: MigrationState) -> boo
     def get_crawler_tasks(self):
         if self._include_object_permissions:
             for item in StaticListing(self._include_object_permissions, self.object_types()):
-                yield partial(self._crawler_task, item.object_id, item.object_type)
+                yield partial(self._crawler_task, item.object_id, sql.ObjectTypePlural(item.object_type))
```

Comment:
Does this change anything? Or is it "just" better practice to reuse the enum from the sdk?

Reply:
`StaticListing` basically mocks what the listing would return to the crawler. The crawler expects an enum and would crash if a string is returned